pkg/flag/misconf_flags.go

```diff
@@ -170,8 +184,8 @@ func (f *MisconfFlagGroup) ToOptions() (MisconfOptions, error) {
 
 	return MisconfOptions{
 		IncludeNonFailures:      f.IncludeNonFailures.Value(),
-		ResetPolicyBundle:       f.ResetPolicyBundle.Value(),
-		PolicyBundleRepository:  f.PolicyBundleRepository.Value(),
+		ResetChecksBundle:       f.ResetChecksBundle.Value(),
+		ChecksBundleRepository:  f.ChecksBundleRepository.Value(),
```

Comment:
We've added logic for the trivy-db and trivy-java-db tags - #6410.
I think users will expect the same logic for "checks".

Reply:
@DmitriyLewen If I understand correctly, you are referring to versioning the bundles? If so, we already have that for checks: https://github.com/aquasecurity/trivy/blob/8c555939b6b243ef860cd4611791e69d38e7d734/pkg/policy/policy.go#L21-L24

If not, could you explain your idea?

Reply:
I meant similar logic - https://github.com/aquasecurity/trivy/pull/6219#pullrequestreview-1914419291

But we do that for backward compatibility:
https://github.com/aquasecurity/trivy/blob/7c2017fa7ad43b310ce487072ace269ed72e8c4a/pkg/flag/db_flags.go#L152-L162

I thought about using the same logic for similar flags, but it looks like you were only adding tag ("BundleVersion") for default repository, so those changes might break user's scanning.

Reply:
OK can we make that change (if needed) in a separate PR?

Reply:
To be honest, I already doubt the need for these changes. We made them for backward compatibility and will probably remove them soon.
So let's leave it to your decision. We can always add this if users ask about it.